games-simulation/simutrans/simutrans-120.2.0.recipe

```diff
@@ -0,0 +1,82 @@
+SUMMARY="A transport simulation game"
+DESCRIPTION="Simutrans is an open-source transportation simulation game \
+available for Windows, Mac, Linux and BeOS. Your goal is to establish a \
+successful transport company and build the necessary infrastructure to \
+transport passengers, mail and goods between cities, factories and \
+touristic attractions by land, air and water."
+HOMEPAGE="http://www.simutrans.com"
+SvnRevision="8077"
+GitRevision="ea08f1eed3fd8ffdd017a522898450047f83d768"
+SOURCE_URI="https://github.com/aburch/simutrans/archive/$GitRevision.tar.gz"
+CHECKSUM_SHA256="96dca621cde48466b38fa6cf7116e18c72d8c6cadca6733cfdc1f2a4048d2e6b"
+SOURCE_FILENAME="simutrans-$portVersion.tar.gz"
+SOURCE_DIR="simutrans-$GitRevision"
+COPYRIGHT="1997-2004 Hj. Malthaner
+	2005-2017 The Simutrans Team"
+LICENSE="Artistic"
+REVISION="1"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+PROVIDES="
+	simutrans$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+	lib:libsdl_mixer$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	simutrans_pakset$secondaryArchSuffix >= 120.2
+	timgmsoundfont
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libbz2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:git
+	cmd:svn
+	cmd:curl
```

Comment:
curl: It is included because the actual language files are downloaded via a script (get_lang_files.sh) during the build process.
svn: Without svn the build process used to stop when it tried to determine the svn revision (of course, with haikuporter it doesn't really get the correct number but then it includes a "0" - it's no problem als long as it can use "svnversion" in the first place). It might be better to use ("cmd:svnversion) in this case?